crates/flux-rustc-bridge/src/lowering.rs

```diff
@@ -789,14 +789,14 @@ impl<'tcx> Lower<'tcx> for rustc_ty::ValTree<'tcx> {
 
     fn lower(self, _tcx: TyCtxt<'tcx>) -> Self::R {
         if let Some(scalar_int) = self.try_to_scalar_int() {
-            return crate::ty::ValTree::Leaf(scalar_int);
+            crate::ty::ValTree::Leaf(scalar_int)
```

Comment:
Wait you didn't change this from the previous PR

Reply:
oops -- good catch! looks like it was in 2 places! this and `lower_constant`...